docs/user/alerting/rule-types/geo-rule-types.asciidoc

```diff
@@ -10,24 +10,90 @@
 
 The tracking containment rule alerts when an entity is contained or no longer contained within a boundary. 
 
+In *{stack-manage-app}* > *{rules-ui}*, click *Create rule*, fill in the name and optional tags, then select *Tracking containment*.
+
 [float]
-=== Requirements
-To create a tracking containment rule, the following requirements must be present:
+=== Define the conditions
+
+When you create a tracking containment rule, you must define the conditions that it detects. For example:
+
+[role="screenshot"]
+image::user/alerting/images/alert-types-tracking-containment-conditions.png[Creating a tracking containment rule in Kibana,width=75%]
+// NOTE: This is an autogenerated screenshot. Do not edit it directly.
+
+. Define the entities index
```

Comment:
Should there be a period or comma at the end of this sentence?

Or how about "Define the entities index, which must contain ..."

Reply:
Fixed in 0089bf11be8315399848ffedbe7b757f28ac1a95